src/keys-builder/utils/create-translation.ts

```diff
@@ -36,7 +45,9 @@ function createJson(config: CreateTranslationOptions) {
 }
 
 function createPot(config: CreateTranslationOptions) {
-  const resolved = resolveTranslation(config);
+  const resolved = getConfig().unflat
+    ? flatten(resolveTranslation(config))
+    : resolveTranslation(config);
```

Comment:
Shouldn't the condition be reveresed?
```suggestion
  const resolved = getConfig().unflat
    ? resolveTranslation(config)
    : flatten(resolveTranslation(config));
```

Reply:
We touched this code in the discussion above already: https://github.com/ngneat/transloco-keys-manager/pull/89#discussion_r1051552285.
It's because POT doesn't support nesting, but internally we have a nested structure if `unflat = true`, so we have to flatten it for POT.